backend/src/app/api/add_subscriber.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
```

Comment:
Why separate file for each subscriber API. All subscriber API should be in a single file.